### DISCUSSION TITLE
The "query_id" in Ton smart contract.

### DISCUSSION THREAD

**Discussion by ethsdev** [community/community#135262] [Programming Help] [answered]
### Body

What is query_id in Ton smart contract

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment, marked as answer]
Please see [this comment](https://github.com/orgs/community/discussions/135263#discussioncomment-10290533) for reference to why this discussion was closed and locked. Thanks!
